### ISSUE TITLE
Problem with application.html.erb and <nav> tags

### ISSUE THREAD

**Issue by JESii** [twilson63/html2haml#5]
Hi... 

I recently switched to Haml and used html2haml to convert my existing views.  I also converted my existing application.html.erb file, but ran into a problem. The %ul entries under the %nav entry were not properly indented, so the <nav> element was empty and the <ul> entries were just hanging out on the left of the screen with no special formatting. I'm on haml 3.1.1 and did the conversion a couple of times before I found out what was happening.

I can send samples of the input/output if you want, but it was a very simple erb file; what I got was this:
  %nav.round
  %ul
    %li= link_to "Home", :welcome_index
    %li= link_to "Incident Report", :incident_reports
    %li= link_to "Check Request", '#'
    %li= link_to "Purchase Requisition", '#'
    %li= link_to "Quote Request", '#'
  %section.round
Notice that the %nav and the %ul are at the same indentation level...

**Comment by twilson63** [comment]
Hello,

Thanks again for using html2haml.

If you could send a full html example, I will be glad to test and fix, what
browser are you using?

Thanks

Tom

On Thu, May 19, 2011 at 3:48 PM, JESii <
reply@reply.github.com>wrote:

> Hi...
> 
> I recently switched to Haml and used html2haml to convert my existing
> views.  I also converted my existing application.html.erb file, but ran into
> a problem. The %ul entries under the %nav entry were not properly indented,
> so the <nav> element was empty and the <ul> entries were just hanging out on
> the left of the screen with no special formatting. I'm on haml 3.1.1 and did
> the conversion a couple of times before I found out what was happening.
> 
> I can send samples of the input/output if you want, but it was a very
> simple erb file; what I got was this:
>  %nav.round
>  %ul
>    %li= link_to "Home", :welcome_index
>    %li= link_to "Incident Report", :incident_reports
>    %li= link_to "Check Request", '#'
>    %li= link_to "Purchase Requisition", '#'
>    %li= link_to "Quote Request", '#'
>  %section.round
> Notice that the %nav and the %ul are at the same indentation level...
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/twilson63/html2haml/issues/5

## 

Tom Wilson
Jack Russell Software Company   LLC
1067 Cliffwood Drive
Mount Pleasant, SC  29464
Phone: 843-606-0637
Email: tom@jackhq.com
Web: http://www.jackhq.com
Calendar: http://www.jackhq.com/calendar

**Comment by twilson63** [comment]
I just tried

```
<nav class="round">
<ul>
<li>1</li>
</ul>
</nav>
```

and got

```
%nav.round
  %ul
    %li 1

```

If you can provide a sample, I would love to take a stab at why it is not working.

Thx

Tom

**Comment by JESii** [comment]
Will definitely get you a sample; I was slammed today and need to be on the 
right computer to get them to you, both input and output.

Cheers...jon

----- Original Message ----- 
From: "twilson63" 
reply@reply.github.com
To: jseidel@edpci.com
Sent: Thursday, May 19, 2011 6:55 PM
Subject: Re: [html2haml] Problem with application.html.erb and <nav> tags 
(#5)

**Comment by JESii** [comment]
Hi, Tom... Sorry it took me so long to get back to you. I've attached the two files: input and output from the conversion... let me know what else I can do to help you on this one...

What I did was the following command just now:
    html2haml application.html.erb_save > application.html.haml_save

Thanks...jon

Jon Seidel, CMC
EDP Consulting, Inc. / www.edpci.com / www.4mypasswords.com
Effective Decisions... Priceless!
LinkedIn Profile
Technology that means Business - Since 1979
3373 Guido Street
Oakland, CA 94602
phone: 510-530-6314
fax: 510-531-1522

The Certified Management Consultant (CMC) mark is awarded to consultants who meet the strict standards defined by the Institute of Management Consultants USA (an ISO/IEC 17024:2003 Certifying Body), including examination by their peers, client evaluations, and a written examination evidencing their understanding of the IMC USA's Code of Ethics. The CMC is the only consulting certification that has been ISO-accredited and recognized by the insurance industry as reducing risk.
Check out IMC USA for more information.
